dkg/dkg.go

```diff
@@ -314,7 +317,10 @@ func Run(ctx context.Context, conf Config) (err error) {
 	}
 
 	if !conf.NoVerify {
-		if err := lock.VerifySignatures(); err != nil {
+		eth1Cl := eth1wrap.NewDefaultEthClientRunner(conf.ExecutionEngineAddr)
+		go eth1Cl.Run(ctx)
```

Comment:
This will run the second instance of the same client. The first one is started by `loadDefinition()` call (see disk.go).
I think for dkg we need to create+start a single instance of the client and run it.
Please verify other usages as well.